vtex/styleguide
# [WIP] 🧪 Experimental Table

#### What is the purpose of this pull request?
WIP

#### What problem is this solving?
Related issue #716 

#### How should this be manually tested?
See in action on [this PR](https://github.com/vtex/admin-collections/pull/48)

You can also clone the repository, start the styleguidist using `cd styleguide && yarn install && yarn start` and navigate to `Experimental/Table` section.

#### Screenshots or example usage
Let's suppose that you have a custom component called `AwesomeComponent` that should have a field with autocomplete. 

The code would be something like:

```js
import { Table, useAutocomplete } from 'vtex.styleguide'
import  AwesomeComponent from './'

function YourUseCase(props) {

  const shema = { ... }
  const items = { ... }

  const density = {
    buttonLabel: 'Line Density',
    lowOptionLabel: 'Low',
    mediumOptionLabel: 'Medium',
    highOptionLabel: 'High',
  }

 const fields = {
    label: 'Toggle Visible Fields',
    showAllLabel: 'Show All',
    hideAlllLabel: 'Hide All',
  }
  
  const inputSearch  = useAutocomplete(...)

  return (
    <Table schema={schema} items={items} toolbar={inputSearch, density, fields} />
  )
}
```

#### Types of changes
